TAA/TAA.md

```diff
@@ -0,0 +1,141 @@
+# Transaction Author Agreement V2
+```
+                             `-::-`                      
+                      ```   `//////`   ```               
+                    .////:` `//////` `:////.             
+                    //////-  `-::-`  -//////             
+                    .:////`    --    `////:.             
+                ....   ` .:`   --   `:. `   ....         
+              `/////:     `:.  ..  .:`     :////:`       
+              .//////-.`    .  ``  .    `.-//////.       
+               .:::````.---  -////-  ---.```::::.        
+                            .//////.                     
+               .-::-` `.---  -////-  ---.` `-::-.        
+              .//////-.`    `  ``  .    `.-//////.       
+              `/////:`    `:.  ..  .:`    `://///`       
+                .--.     .:`   --   `:.     .--.         
+                    .:////`    --    `////:.             
+                    //////-  `.::.`  -//////             
+                    .////:` `//////` `:////.             
+                      ```   `//////`   ```               
+                             `-::-`                      
+                                                                    
+ ssSSSSSss                                         II               
+sS"     "Ss                                        ""               
+SS,                                                                 
+`SSsssss,     ,ooOOOoo,   Vv       vV  Rr,rRRRrr,  II  Nn,nNNNnn,   
+  `"""""Ss,  oO"     "Oo  `Vv     vV'  RRR'   "RR  II  NNN'   `"Nn  
+        `Ss  OO       OO   `Vv   vv'   RR          II  NN       NN  
+SSs     SSS  "Oo,   ,oO"    `Vv,vV'    RR          II  NN       NN  
+ "SSSSSSS"    `"OoooO"'       "V"      RR          II  NN       NN  
+```
+[https://sovrin.org/](https://sovrin.org/)
+## **Summary:**
+* This summary is provided to help you understand your obligations when writing to the Sovrin Ledger Networks-it does not have any legal effect or replace the full legal text of the agreement provided below it.
```

Comment:
> when writing to the Sovrin Ledger Networks-it does ...

I think the following would be more readable:
> when writing to the Sovrin Ledger Networks -- it does ...

Reply:
I agree. @lynnbendixsen what is your opinion?

Reply:
I agree also.  The double hyphen appears to be the most grammatically correct also.